elasticsearch/README.md

```diff
@@ -91,6 +91,8 @@ helm install --name elasticsearch elastic/elasticsearch --version 7.2.0 --set im
 | `httpPort`                 | The http port that Kubernetes will use for the healthchecks and the service. If you change this you will also need to set [http.port](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-http.html#_settings) in `extraEnvs`                                                                          | `9200`                                                                                                                    |
 | `transportPort`            | The transport port that Kubernetes will use for the service. If you change this you will also need to set [transport port configuration](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-transport.html#_transport_settings) in `extraEnvs`                                                        | `9300`                                                                                                                    |
 | `service.type`             | Type of elasticsearch service. [Service Types](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)                                                                                                                                                                         | `ClusterIP`                                                                                                               |
+| `serviceAccount.create`    | Create service account               | `true`                                                                                                                                   |
```

Comment:
I didn't leave a comment on this PR since I left it on the Kibana PR (since they were opened at the same time both by you). But I think we should also rename this one to match with the suggestion for Kibana which matches filebeat and metricbeat too https://github.com/elastic/helm-charts/pull/208#pullrequestreview-258946623

Reply:
Sounds good! Sorry for dropping the ball on this a little, I'll get to it in the coming days!

Reply:
Please see https://github.com/elastic/helm-charts/pull/208#discussion_r311604824

Reply:
Seeing that this chart never had a serviceaccount, can we keep this one as is?

Reply:
That would be fine with me. Consistency between the charts would be nicer though.